app/src/androidTest/java/com/hedvig/app/feature/embark/multiaction/MultiActionTest.kt

```diff
@@ -0,0 +1,146 @@
+package com.hedvig.app.feature.embark.multiaction
+
+import android.view.KeyEvent
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.matcher.RootMatchers
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.espresso.matcher.ViewMatchers.hasErrorText
+import androidx.test.espresso.matcher.ViewMatchers.isRoot
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import com.agoda.kakao.common.views.KView
+import com.hedvig.android.owldroid.graphql.EmbarkStoryQuery
+import com.hedvig.app.feature.embark.ui.EmbarkActivity
+import com.hedvig.app.testdata.feature.embark.data.STORY_WITH_MULTI_ACTION
+import com.hedvig.app.util.ApolloCacheClearRule
+import com.hedvig.app.util.ApolloMockServerRule
+import com.hedvig.app.util.LazyActivityScenarioRule
+import com.hedvig.app.util.apolloResponse
+import com.hedvig.app.util.context
+import com.kaspersky.kaspresso.testcases.api.testcase.TestCase
+import org.junit.Rule
+import org.junit.Test
+
+class MultiActionTest : TestCase() {
+    @get:Rule
+    val activityRule = LazyActivityScenarioRule(EmbarkActivity::class.java)
+
+    @get:Rule
+    val apolloMockServerRule = ApolloMockServerRule(
+        EmbarkStoryQuery.QUERY_DOCUMENT to apolloResponse { success(STORY_WITH_MULTI_ACTION) }
+    )
+
+    @get:Rule
+    val apolloCacheClearRule = ApolloCacheClearRule()
+
+    @Test
+    fun errorInput() = run {
+        activityRule.launch(EmbarkActivity.newInstance(context(), this.javaClass.name, ""))
+
+        step("Press add building to display bottom sheet") {
+            MultiActionScreen {
+                multiActionList {
+                    childAt<MultiActionScreen.AddBuildingButton>(0) {
+                        click()
+                    }
+                }
+            }
+        }
+
+        step("Input more than max value") {
+            AddBuildingBottomSheetScreen {
+                dropDownMenu {
+                    click()
+                }
+
+                onView(withText("Attefall"))
+                    .inRoot(RootMatchers.isPlatformPopup())
+                    .perform(click());
+
+                numberInput {
+                    typeText("15000")
+                }
+
+                numberLayout {
+                    hasError("Max input")
```

Comment:
Is this correct? 🤔 From where does this text come from?

Reply:
We need string resources for this, I'll ask Zak